packages/teams-js/src/public/pages.ts

```diff
@@ -240,23 +240,24 @@ export namespace pages {
 
     /**
      * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
-     * back to the App via the Context object.
+     * back to the application via the Context object.
      */
     subPageId?: string;
 
     /**
-     * Optional ID of the Teams Channel where the app should be opened
+     * Optional ID of the Teams Channel where the application should be opened
      */
     channelId?: string;
   }
 
   /**
-   * Namespace to interact with the teams specific part of the SDK.
+   * Provides APIs for querying and navigating between an application's "tabs", which are pages associated
```

Comment:
I welcome any additional improvements that can be made to this description.

Reply:
Here's a stab at editing out the quotes:

Provides APIs for querying and navigating between contextual tabs of an application. Unlike personal tabs, contextual tabs are pages associated with a specific context, such as channel or chat.

Reply:
It's so beautiful! It's much, much better, thank you!

Reply:
Any reason why "tabs" in is quotes? also not sure what, "which are pages associated" means?
Maybe its just "Provides APIs for querying and navigating between an application's tabs"?